tests.py

```diff
@@ -679,6 +679,79 @@ def test_bookmarks_dont_return_link_last_when_on_last(self):
 
         self.assertTrue('Link' not in res.headers)
 
+    @single_user
+    def test_add_bookmark_with_tag(self):
+        bookmark = {
+            'url': 'http://github.com',
+            'title': 'test title',
+            'tags': ['github', 'test'],
+        }
+        res = self.add_bookmark(bookmark)
+        self.assertEqual(HTTPStatus.CREATED, res.status_code)
+
+    @single_user
+    def test_get_bookmark_with_tag(self):
+        bookmark = {
+            'url': 'http://github.com',
+            'title': 'test title',
+            'timestamp': '2016-11-06T01:31:15',
+            'tags': ['github', 'test'],
+        }
+        res = self.add_bookmark(bookmark)
+        self.assertEqual(HTTPStatus.CREATED, res.status_code)
+
+        res = self.app.get(
+            '/api/v1/bookmarks',
+            headers=[self.user1])
+        self.assertEqual(HTTPStatus.OK, res.status_code)
+
+        result = get_json(res)
+        self.assertEqual([dict(bookmark, id=1, read=None)], result)
```

Comment:
A test with one assertion (or unique to other tests assertions) which folows [AAA](http://wiki.c2.com/?ArrangeActAssert) pattern is more readable. Say,
```python
    def test_get_bookmark_with_tag(self):
         bookmark = {
             'url': 'http://github.com',
             'title': 'test title',
             'timestamp': '2016-11-06T01:31:15',
             'tags': ['github', 'test'],
         }

         res = self.app.get(
             '/api/v1/bookmarks',
             headers=[self.user1])
  
         self.assertEqual(HTTPStatus.OK, res.status_code)
         self.assertEqual([dict(bookmark, id=1, read=None)], get_json(res))
```

Reply:
:+1: